audio_capture_wave.launch crashes on Ubuntu 16.04 - Kinetic

$ roslaunch audio_capture capture_wave.launch 
    ... logging to /home/rein/.ros/log/922db3c8-e2e9-11e6-99bb-a0a8cd88ac07/roslaunch-athome10-21614.log
    Checking log directory for disk usage. This may take awhile.
    Press Ctrl-C to interrupt
    Done checking log file disk usage. Usage is <1GB.
    
    started roslaunch server http://athome10:45245/
    
    SUMMARY
    ========
    
    PARAMETERS
     * /audio_capture/channels: 1
     * /audio_capture/depth: 16
     * /audio_capture/format: wave
     * /audio_capture/sample_rate: 16000
     * /rosdistro: kinetic
     * /rosversion: 1.12.2
    
    NODES
      /
        audio_capture (audio_capture/audio_capture)
    
    ROS_MASTER_URI=http://athome10:11311
    
    core service [/rosout] found
    process[audio_capture-1]: started with pid [21974]
    
    (audio_capture:21974): GStreamer-WARNING **: 0.10-style raw audio caps are being created. Should be audio/x-raw,format=(string).. now.
    [ERROR] [1485341596.863962893]: Unsupported media type.
    terminate called after throwing an instance of 'boost::exception_detail::clone_impl<boost::exception_detail::error_info_injector<boost::lock_error> >'
      what():  boost: mutex lock failed in pthread_mutex_lock: Invalid argument
    [audio_capture-1] process has died [pid 21974, exit code -6, cmd /opt/ros/kinetic/lib/audio_capture/audio_capture __name:=audio_capture __log:=/home/rein/.ros/    log/922db3c8-e2e9-11e6-99bb-a0a8cd88ac07/audio_capture-1.log].
    log file: /home/rein/.ros/log/922db3c8-e2e9-11e6-99bb-a0a8cd88ac07/audio_capture-1*.log
    all processes on machine have died, roslaunch will exit
    shutting down processing monitor...
    ... shutting down processing monitor complete
    done

--- a/audio_capture/src/audio_capture.cpp
+++ b/audio_capture/src/audio_capture.cpp
@@ -100,7 +100,7 @@ namespace audio_transport
           link_ok = gst_element_link_many(_source, _filter, _convert, _encode, _sink, NULL);
         } else if (_format == "wave") {
           GstCaps *caps;
-          caps = gst_caps_new_simple("audio/x-raw-int",
+          caps = gst_caps_new_simple("audio/x-raw",
                                      "channels", G_TYPE_INT, _channels,
                                      "width",    G_TYPE_INT, _depth,
                                      "depth",    G_TYPE_INT, _depth,

